webpack.config.js

```diff
@@ -53,10 +53,12 @@ const browserConfig = env => {
                         {
                             loader: 'sass-loader',
                             options: {
-                                includePaths: [
-                                    path.resolve(__dirname, 'node_modules'),
-                                    path.resolve(__dirname, './src/styles'),
-                                ]
+                                sassOptions: {
+                                    includePaths: [
+                                        path.resolve(__dirname, './node_modules'),
+                                        path.resolve(__dirname, './src/styles'),
```

Comment:
technically this should be `path.resolve(__dirname, 'src', 'styles')`, but that's a problem for another commit.